index.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const {
+  getAllChampions,
+  getChampionsByIdWithRegionAndRoles,
+  postChampion,
+  deleteChampion
+} = require('./controllers/champions')
+const { getAllRegions, getRegionByIdWithChampionsAndRoles } = require('./controllers/regions')
+const { getAllRoles, getRoleByIdWithChampionsAndRegions } = require('./controllers/roles')
+
+const app = express()
+
+app.set('view engine', 'pug')
+app.use(express.static('public'))
+
+app.get('/lol', (request, response) => {
+  return response.render('index')
+})
```

Comment:
A user wouldn't know to navigate to this route without having seen your code, they'd go to the root route, see this, and assume it was broken.

![image](https://user-images.githubusercontent.com/5775716/84073715-5342c380-a99f-11ea-934d-38109c6f08c6.png)